www/index.html

```diff
@@ -163,21 +163,22 @@ <h3 class="track-title">Arts</h3>
 				<a target="_blank" href="http://www.embeddedplanet.com"><img id="sponsors_eb" src="sponsor-logos/embedded-planet.jpg"></a>
 				<a target="_blank" href="https://www.ibm.com/watson/health/explorys/"><img id="sponsors_explorys" src="sponsor-logos/Explorys.svg"></a>
 			</div>
-			<div style="text-align:center; font-size:22px;">
-				<p>Learn more about sponsoring HackCWRU<br>Interested?  Email <a class="email_link" href="mailto:sxc868@case.edu">Sacheth</a>!</p>
-			</div>
 		</div>
-		<footer>
-			<div id="contact" class="footer">
-				<div class="copyr">
+		<div id="contact">
```

Comment:
Yes, namely in terms of accessibility for various screen readers, and having "semantically correct" HTML

[This blog post](http://www.hongkiat.com/blog/html-5-semantics/) is a pretty decent introduction to semantic HTML, with some links to reasons why you should care about it, if you're looking for further reading